### PR TITLE
Avoid unintentionally change spinbox's and comboBox's value in "Prope…

### PR DESCRIPTION
**Pull request by hackery98**
#1600

### CODE REVIEW

File: qCC/db_tree/ccPropertiesTreeDelegate.cpp [hunk 1]
```diff
@@ -2341,6 +2368,7 @@ void ccPropertiesTreeDelegate::octreeDisplayedLevelChanged(int val)
 		octree->setDisplayedLevel(val);
 		updateDisplay();
 		//we must also reset the properties display!
+		m_lastFocusItemRole = OBJECT_OCTREE_LEVEL;
```
Comment (dgirardeau) [inline qCC/db_tree/ccPropertiesTreeDelegate.cpp:2371]: I guess these lines shouldn't be below the comment which applies to the call to 'updateModel()'. Can you move them after updateModel() or before the comment?

Comment (hackery98) [reply qCC/db_tree/ccPropertiesTreeDelegate.cpp:2371]: > 		//record item role here to use it 'setFocus' in 'createEditor', 
> 		//than you can continuous scroll the widget's value.
> 		m_lastFocusItemRole = OBJECT_CONE_HEIGHT;
> 
> 		//we must also reset the properties display!
> 		updateModel();
is this ok? Please let me know if there is a problem with the expression .

Comment (dgirardeau) [reply qCC/db_tree/ccPropertiesTreeDelegate.cpp:2371]: Thanks for asking. I would simply write something like:
//record item role to force the scroll focus (see 'createEditor').

**Review by dgirardeau — APPROVED** [review]

**Comment by dgirardeau** [comment]
Thanks for the contribution

**Pull request merged**